slides/03_traitsui.md

```diff
@@ -0,0 +1,262 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.7
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+<!-- #region slideshow={"slide_type": "slide"} -->
+## Sharing scientific tools: script to desktop application
+
+### TraitsUI
+
+**Jonathan Rocher, Siddhant Wahal, Jason Chambless, Corran Webster, Prabhu Ramachandran**
+
+**SciPy 2022**
+
+<!-- #endregion -->
+
+<!-- #region slideshow={"slide_type": "slide"} -->
+## TraitsUI: Easy GUI building
```

Comment:
Still need to digest these slides, but here's the list of TraitsUI concepts we'll need:
- Stage 3:
    - Item
        - name
        - show_label
        - title
    - View
        - `View` trait for `HasStrictTraits`
        - resizable
        - width
        - buttons
    - OKButton
    - Difference between `configure_traits` and `edit_traits`
    - ModelView
    - Editors and MplFigureEditor
- Stage 4.0:
    - InstanceEditor
    - Style?
- Stage 4.1:
    - FileEditor
    - Editor styles
        - custom
    - edit_traits:
        - kind
        - parent
- Stage 5.1
    - Item
        - visible_when
    - Editor styles:
        - readonly
    - HGroup, Item, Label, ModelView, Spring, View
    - DataFrameEditor
    - Item
        - visible_when
- Stage 5.2:
    - enabled_when
- Stage 5.3:
    - BoolEditor, EnumEditor